.github/workflows/cron_php_update_modules.yml

```diff
@@ -21,7 +23,9 @@ jobs:
         run: composer install --prefer-dist
 
       - name: Update PrestaShop packages
-        run: echo "PULL_REQUEST_BODY=$(php .github/workflows/cron_update_modules/script.php)" >> $GITHUB_ENV
```

Comment:
The problem come from that you didn't escape  `\n` 

```
run: echo "PULL_REQUEST_BODY='$(php .github/workflows/cron_update_modules/script.php)'" >> $GITHUB_ENV
```
should have been enough (to try)

Reply:
We can try it.

Reply:
Thanks